src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -1,7 +1,44 @@
 package core.basesyntax;
 
+import java.time.LocalDate;
+
 public class SalaryInfo {
+    static final String HEADER = "Report for period ";
+    static final String DELIMITER = " ";
+    static final String SEPARATOR = " - ";
+    static final String DOT = "\\.";
+    static final String DASH = "-";
+    static final int ZERO = 0;
+    static final int ONE = 1;
+    static final int TWO = 2;
+    static final int THREE = 3;
```

Comment:
Consider giving meaningful names to your constants. 
Naming constants just by their value is a missed opportunity to make the code more self-explanatory.
The name of a constant should indicate its role or purpose within the application, providing context to other developers.

Instead of:
// private static final int ZERO = 0;
// private static final int FIVE = 5;
// private static final String DASH = "-";

Consider something like:
// private static final int AMOUNT_INDEX = 0;
// private static final int DEFAULT_VALUE = 5;
// private static final Strint DELIMITER = "-";